index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>My Portfolio</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+
+</head>
+
+<body>
+    <header class="container">
+        <nav class="nav-item1">My Logo
+        </nav>
+        <nav class="nav-item2">
+        </nav>
+        <nav class="nav-item3">
+            <img src="images/toggle.png" alt="toggle">
+        </nav>
+    </header>
+    <main class="content">
+            <h1 class="box1">I’m Emmanuel
```

Comment:
[OPTIONAL]
Following HTML & CSS best practices, it would be nice to use the `section` tag for the headline section. The portfolio will have many sections in the future. 😀